test/dotnet-new3.UnitTests/DotnetNewInstantiate.Approval.cs

```diff
@@ -307,5 +307,33 @@ public Task CanShowWarning_WhenHostDataIsIncorrect()
             });
         }
 
+        [Fact]
+        public Task CanShowError_OnTemplatesWithSameShortName()
+        {
+            string home = TestUtils.CreateTemporaryFolder("Home");
+            string workingDirectory = TestUtils.CreateTemporaryFolder();
+            string templateLocation = Helpers.InstallTestTemplate("Invalid/SameShortName", _log, home, workingDirectory);
+
+            var commandResult = new DotnetNewCommand(_log, "sameshortname")
+                .WithCustomHive(home)
+                .WithWorkingDirectory(workingDirectory)
+                .Execute();
+
+            commandResult
```

Comment:
nit: Why not keep this as one fluent chain? (debugging relict?)

Reply:
we need to pass `commandResult` to `Verifier` below. 
Chain will not return it.